implementation/src/main/java/io/smallrye/metrics/exporters/OpenMetricsExporter.java

```diff
@@ -37,17 +35,19 @@ public String exportAllScopes() {
         for (MeterRegistry meterRegistry : prometheusRegistryList) {
             PrometheusMeterRegistry promMeterRegistry = (PrometheusMeterRegistry) meterRegistry;
             //strip "# EOF"
-            sb.append(promMeterRegistry.scrape(TextFormat.CONTENT_TYPE_OPENMETRICS_100).replaceFirst("\r?\n?# EOF", ""));
+
+            String scraped = promMeterRegistry.scrape(TextFormat.CONTENT_TYPE_OPENMETRICS_100).replaceFirst("# EOF\r?\n?", "");
```

Comment:
The value of the `scraped` variable is being ignored, so the output from this function is empty. I've corrected this in my local copy, and it appears to work ok.

Reply:
Looks like this change was introduced here: https://github.com/smallrye/smallrye-metrics/commit/2e2d548744b525786e32a45f9328dee7eba24b33#diff-f7618f122be0f8b181e0c9f6a521fab33a07461c95fd5a2e1b03981d6b40641eR39

Reply:
Ah, yes! sorry!  Forgot to re-rebase. pushed it now.